pom.xml

```diff
@@ -287,5 +287,25 @@
        <url>https://repo.jenkins-ci.org/public/</url>
      </pluginRepository>
   </pluginRepositories>
+
+  <profiles>
+    <profile>
+      <id>java8-doclint-disabled</id>
```

Comment:
This is the 2.6.x branch. All failing here. Didn't need this for the master branch